docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -124,29 +124,105 @@ You can attach more than one action. Clicking the *Add action* button will promp
 [float]
 [[defining-rules-actions-variables]]
 ===== Action variables
-Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass rule values at the time a condition is detected to an action. You can access the list of available variables using the "add rule variable" button. Although available variables differ by rule type, all rule types pass the following variables:
 
+Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass rule values to an action at the time a condition is detected. You can access the list of available variables using the "add rule variable" button.
+
+All rule types pass the following variables:
+
+`date`:: The date the rule scheduled the action, in ISO format.
+`kibanaBaseUrl`:: The configured <<server-publicBaseUrl,`server.publicBaseUrl`>>. If not configured, this will be empty.
 `rule.id`:: The ID of the rule.
 `rule.name`:: The name of the rule.
 `rule.spaceId`:: The ID of the space for the rule.
 `rule.tags`:: The list of tags applied to the rule.
-`date`:: The date the rule scheduled the action, in ISO format.
-`alert.id`:: The ID of the alert that scheduled the action.
+
+There is also a set of action variables specific to the action frequency:
+
+- <<alert-action-variables,For each alert>>
```

Comment:
Are we also updating the docs to describe the new summary notification dropdown?

Reply:
Do you think the "Action variable context" section should be moved up closer to this "For each alert" section? Summary notifications don't have context variables

Reply:
> Are we also ... summary notification dropdown?

Yes, I think it's okay to do that in a separate PR, as part of the larger updates and screenshot refreshes related to this feature.

Reply:
> ... Summary notifications don't have context variables.

I think it's okay to leave those sections where they are for now, but we should update the intro to make it clear that they don't exist in the summary case.